packages/tooling/fast-tooling-react/package.json

```diff
@@ -114,6 +114,8 @@
     "@microsoft/fast-components": "^1.21.8",
     "@microsoft/fast-components-class-name-contracts-base": "^4.8.0",
     "@microsoft/fast-components-foundation-react": "^3.2.0",
+    "@microsoft/fast-element": "^1.1.0",
```

Comment:
@janechu should these target the latest version so that the deps are symlinked?

Reply:
They should, looks like maybe that one got missed in the rebase.

Reply:
@nicholasrice @janechu I can take these if they haven't been addressed yet

Reply:
Resolved!